packages/batch-submitter/src/transaction-chain-contract.ts

```diff
@@ -38,17 +62,21 @@ const appendSequencerBatch = async (
   batch: AppendSequencerBatchParams,
   options?: TransactionRequest
 ): Promise<TransactionResponse> => {
-  const methodId = keccak256(
-    Buffer.from(APPEND_SEQUENCER_BATCH_METHOD_ID)
-  ).slice(2, 10)
-  const calldata = encodeAppendSequencerBatch(batch)
   return OVM_CanonicalTransactionChain.signer.sendTransaction({
     to: OVM_CanonicalTransactionChain.address,
-    data: '0x' + methodId + calldata,
+    data: getEncodedCalldata(batch),
     ...options,
   })
```

Comment:
Is this method now unused?

Reply:
Oh wow yes it is! I'm thinking it could make sense to leave it in in case we ever tried to call `appendSequencerBatch` and it fails because it doesn't accept any parameters. But you can let me know if you want me to just delete it -- I'm happy to